pyaerocom/_logging.py

```diff
@@ -12,12 +12,13 @@
 from __future__ import annotations
 
 import logging
+import os
 from importlib import resources
 from logging.config import fileConfig
 
 LOGGING_CONFIG = dict(
     # root logger
-    file_name="pyaerocom.log",
+    file_name=os.getenv("PYAEROCOM_LOG_FILE", "pyaerocom.log"),
```

Comment:
personally I would prefer to set the entire log file name via the env variable. Just like many command line tools support.

Reply:
you can provide full path on `PYAEROCOM_LOG_FILE`

Reply:
ups, I just realised that I did not understand the `os.getenv` method. On the other hand the Python documentation mentions this syntax (which I find clearer): `os.getenv(key, default=None)`
Would you mind adding the default keyword?

Reply:
done